developers/bindings/index.md

```diff
@@ -929,7 +929,47 @@ It uses the `getThingUID` method to create the thing UID of the result.
 
 ### Discovery that is bound to a Thing
 
-TODO
+When the discovery process is dependent on a configured bridge the discovery service must be bound to the bridge handler.
```

Comment:
I suggest to use a different wording here. Currently it sounds like a mandatory design pattern but imo it's not.

Why? This way of implementation has a limitation I struggled into when trying to apply it to e.g. the OWM binding. The `AbstractDiscoveryService` for example provides access to other OSGi services (providers) like `Transtlation` and `Locale`. IIRC there's no way to inject those services when using the `ThingHandlerService` pattern. The OHC implementation doesn't take care of that.